Tests/FuzzilliTests/WeakSetTest.swift

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import XCTest
+@testable import Fuzzilli
+
+class WeakSetTests: XCTestCase {
+
+    func testWeakSetInsert() {
```

Comment:
Maybe there should be another test that inserts 2 or more separate objects into the set and then all or some of them become `nil` to check that this works correctly (just because a set is normally not supposed to contain two identical values, but I guess in that case it would?)